src/main/webapp/queue.js

```diff
@@ -44,6 +64,53 @@ function updateShareTab() {
       'lounge.html/?room_id='+ roomParameters +'" type="text" readonly</input>';
 }
 
+function voteToSkip() { // eslint-disable-line no-undef
+  voteBtnCount++;
+  if (voteBtnCount % 2 == 1) {
+    db.collection('rooms') // eslint-disable-line no-undef
+        .doc(roomParameters)
+        .collection('CurrentVideo')
+        .doc('PlaybackData')
+        .update({
+          skipVotes: firebase.firestore.FieldValue.increment(1), // eslint-disable-line no-undef
+        }).then(function() {
+          console.log('votes incremented');
+        }).catch(function(error) {
+          console.log('voting caused an error: ', error);
+        });
+  } else if (voteBtnCount % 2 == 0) {
+    db.collection('rooms') // eslint-disable-line no-undef
+        .doc(roomParameters)
+        .collection('CurrentVideo')
+        .doc('PlaybackData')
+        .update({
+          skipVotes : firebase.firestore.FieldValue.increment(-1),
+        }).then(function() {
+          console.log('votes decremented');
+        }).catch(function(error) {
+          console.log('voting caused an error: ', error);
+        });
+  }
+  setTimeout(2000);
+}
+
+function resetSkips() { // eslint-disable-line no-undef
+  voteBtnCount = 0;
+  db.collection('rooms') // eslint-disable-line no-undef
+      .doc(roomParameters)
+      .collection('CurrentVideo')
+      .doc('PlaybackData')
+      .update({
+        skipVotes: 0,
+        videoId: '',
+      }).then(function() {
+        switchDisplay();
```

Comment:
Forgot to mention this on the call, but you'll also have to reset the playback information so the next video doesn't start at the timestamp where the last one left off.

Reply:
If it's easier, since my reset already sets videoId to an empty string and the playback info to a starting position, you can just add skipVotes:0 to resetPlaybackInfo() in IFrame.js and call that.